superduperdb/base/config.py

```diff
@@ -103,6 +110,7 @@ class Server(JSONable):
     host: str = '127.0.0.1'
     port: int = 3223
     protocol: str = 'http'
+    vector_search: VectorSearchServer = Factory(VectorSearchServer)
```

Comment:
Let us do the configuration via `uri` only (in base config).

Reply:
done